### DISCUSSION TITLE
Homepage on Github Pages empty

### DISCUSSION THREAD

**Discussion by niritum** [mmistakes/minimal-mistakes#3132] [Q&A]
Git repository URL: https://github.com/niritum/niritum.github.io
GitHub Pages hosted: https://niritum.com

My homepage is empty on Github Pages despite it working without any errors on local PC.
Used "Gem-based method".  What did I do wrong?

**Comment by iBug** [comment]
Gem-based method does not "automatically work" on GitHub Pages. You need to setup some CI building or use remote-theme method (which is GH Pages-compatible out-of-the-box).
